scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -2521,11 +2521,22 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
               val trm =
                 if (token.is[Indentation.Indent]) blockExpr()
                 else if (location != BlockStat) expr()
-                else block()
+                else
+                  autoPos {
+                    if (token.is[LeftBrace]) {
+                      Term.Block(inBraces(blockStatSeq()))
+                    } else {
+                      blockStatSeq() match {
+                        case (head: Term) :: Nil => head
```

Comment:
should we keep it as block? who cares if it's just one statement. generally allows one to understand when blocks are expected in the code.

Reply:
I would not add a block if {} are not used I think. This was also done on purpose before.

Reply:
i will leave it to you. in scalafmt, we can mostly adjust.

but if you asked me, i would use it whenever it's expected (in `if`, `case`, `def` etc.), regardless whether it's one statement, and whether it had braces. not using a block with a single statement and/or no braces is akin to not using a template if the body contains just one statement and no parents.

if you _had_ kept the block, solving the long-standing scalafmt request to add back the braces would have been a lot easier.

Reply:
I will leave as is for now, I don't really want to change the base behaviour, since that can have unindented consequences,